products/mysql.md

```diff
@@ -27,9 +27,9 @@ auto:
 releases:
 -   releaseCycle: "8.0"
     latest: '8.0.32'
-    support: 2023-04-30
+    support: 2025-04-30
     eol: 2026-04-30
-    latestReleaseDate: 2022-12-16
+    latestReleaseDate: 2023-01-17
```

Comment:
The latest release date is updated automatically from the git tag date on https://github.com/mysql/mysql-server/tags (so it will be reverted tomorrow). This is not the real date, but it's OK IMO because the date is usually pretty close to the real one and it does not require any manual intervention.